test/models/usergroup_test.rb

```diff
@@ -314,82 +314,88 @@ def populate_usergroups
 
   context 'audit usergroup' do
     context 'child usergroups' do
-      setup do
-        @usergroup = FactoryBot.create(:usergroup, :with_auditing)
-        @child_usergroup = FactoryBot.create(:usergroup)
-        @usergroup.usergroup_ids = [@child_usergroup.id]
-        @usergroup.save
-      end
+      let (:usergroup) { FactoryBot.create(:usergroup, :with_auditing) }
+      let (:child_usergroup) { FactoryBot.create(:usergroup) }
 
       test 'should audit when a child-usergroup is assigned to a parent-usergroup' do
-        recent_audit = @usergroup.audits.last
+        usergroup.usergroup_ids = [child_usergroup.id]
+        usergroup.save
```

Comment:
Theese two lines can still stay in the `setup` block.
Or rather in the `before` block (the same meaning, but sounds better IMHO).